proto/src/main/proto/Rss.proto

```diff
@@ -358,7 +359,13 @@ message FetchRemoteStorageRequest {
   string appId = 1;
 }
 
+message RemoteStorageConfItem {
+  string key = 1;
+  string value = 2;
+}
+
 message FetchRemoteStorageResponse {
   StatusCode status = 1;
   string remoteStorage = 2;
+  repeated RemoteStorageConfItem remoteStorageConf = 3;
```

Comment:
It's better to have RemoteStorage:
message RemoteStorage {
   string path;
   repeated RemoteStorageConfItem remoteStorageConf = 3;
}

Reply:
sure